src/gridstack.ts

```diff
@@ -2661,7 +2669,10 @@ export class GridStack {
       this._updateContainerHeight();
 
       const target = event.target as GridItemHTMLElement;// @ts-ignore
-      this._writePosAttr(target, node);
+      // Do not write sidebar item attributes back to the original sidebar el
+      if (!node._sidebarOrig) {
+        this._writePosAttr(target, node);

```

Comment:
good fix to not write sidebar back, but still seems ,like we need to write to the dropped item... will have to debug.